content/news/preview-20190606.md

```diff
@@ -0,0 +1,70 @@
+It has been quite a while since our last release, but not to worry, we are still going - mostly rewriting tons of code not visible to the outside eye. While not presenting a full playtest we want to take the time to introduce the community to some upcoming changes to look forward to.
+
+### Singleplayer savegames
+
+Probably the most wanted missing feature, the next version will include support for savegames in both singleplayer and the campaign. No more losing all progress to a stray dog eating your Tanya!
```

Comment:
"in both _skirmish_ and the campaign**s**". Campaigns are what people usually think of first when talking about 'singleplayer'. Also plural campaigns, we have more than one, afterall.